zerver/data_import/import_util.py

```diff
@@ -543,6 +543,7 @@ def build_attachment(
         is_realm_public=True,
         path_id=s3_path,
         file_name=fileinfo["name"],
+        content_type=fileinfo.get("mimetype"),
```

Comment:
Do we need any validation/restrictions on what content types are permitted here? I guess that should ideally happen not here, but in `import_realm.py`, to the extent it is required.

Reply:
We don't restrict what content-types people can upload via Zulip itself -- we accept whatever their browser says.  Since we serve this same content-type back when we serve the attachment, we are careful about which content-types we serve as inline.  But we don't limit what types we will serve, or accept.